packages/manager/modules/exchange/src/account/add/account-add.html

```diff
@@ -6,366 +6,785 @@
     <form
         data-ng-if="!$ctrl.isFetchingCreationOptions"
         name="$ctrl.newAccountForm"
+        class="container"
     >
         <oui-back-button
             data-on-click="$ctrl.messaging.resetMessages(); $ctrl.hide()"
-            ><span data-translate="exchange_account_add_back_button"></span
+        ><span data-translate="exchange_account_add_back_button"></span
         ></oui-back-button>
 
         <div
             class="mb-3"
             data-ng-bind=":: 'exchange_account_add_header' | translate"
         ></div>
 
-        <div
-            class="oui-field"
-            data-ng-if="$ctrl.accountCreationOptions.availableTypes.length > 1"
-        >
-            <label
-                class="oui-field__label oui-label"
-                for="accountType"
-                data-ng-bind=":: 'exchange_account_add_accountType_label' | translate"
-            ></label>
-            <div class="oui-field__control oui-field__control_m">
-                <label class="oui-select d-inline-block w-75">
-                    <select
-                        id="accountType"
-                        name="accountType"
-                        class="oui-select__input"
-                        data-ng-model="$ctrl.newAccount.accountType"
-                        data-ng-options="type.displayName for type in $ctrl.accountCreationOptions.availableTypes | orderBy:'displayName' track by type.displayName"
-                        required
-                        aria-describedby="accountTypeDescription"
-                    ></select>
-                    <span
-                        class="oui-icon oui-icon-chevron-down"
-                        aria-hidden="true"
-                    ></span>
-                </label>
-                <span
-                    data-ng-if="$ctrl.exchangeServiceInfrastructure.isDedicatedCluster()"
-                >
-                    <span
-                        id="accountTypeDescription"
-                        class="sr-only"
-                        data-ng-bind=":: 'exchange_account_add_accountType_tooltip' | translate"
-                    ></span>
-                    <span
-                        class="oui-icon ml-4 mt-3 oui-icon-help-circle"
-                        data-oui-tooltip="{{:: 'exchange_account_add_accountType_tooltip' | translate }}"
-                        aria-hidden="true"
-                    ></span>
-                </span>
+        <!--Account info section-->
+        <div class="pl-3">
+            <div class="row">
+                <p class="oui-heading_4 p-0" data-translate="exchange_account_add_personal_account_info_title"></p>
```

Comment:
**issue(blocking):** Only columns (`.col-*-*`) may be children of `.rows`
see: https://github.com/twbs/bootlint/wiki/E013

Reply:
Fixed